modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -110,8 +122,18 @@ If you do not see *Add TLS Context*, ensure you are viewing your secret group in
 . After saving your TLS context, click *Secret Groups*, then click the *Finish*. This enables TLS Context for your *Runtime Fabric.*
 
 == Enable Inbound Traffic
+The Runtime Fabric internal load balancer supports OpenSSL 1.1.1 and TLS 1.3, which provide the following security enhancements and performance improvements over TLS 1.2:
+ 
+* 2x or greater TLS 1.2 Connection Performance Throughput Boost.
```

Comment:
2x compared to what?

If we don't have a good answer, we may want to consider removing.

Reply:
I've added text per Chris.